microsoft-edge/webview2/release-notes.md

```diff
@@ -147,7 +147,149 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 ###### Runtime-only
 ###### SDK-only
 -->
-<!-- end of Jan 2024 Prerelease SDK -->
+<!-- end of Mar. 2024 Prerelease SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2415-prerelease
+
+Release Date: February 21, 2024<!-- todo: update date here & at top of file -->
+
+[NuGet package for WebView2 SDK 1.0.2415-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2415-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2415.0 or higher.<!-- todo: fix version # -->
+
+
+<!-- ------------------------------ -->
+#### Breaking changes
+
+* Change in `LaunchingExternalURISchemeEventArgs` behavior.  If the `InitiatingOrigin` is an [opaque origin](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque),the `InitiatingOrigin` reported in the event args will be its precursor origin.<!-- todo: link for more info -->

```

Comment:
@victorhuangwq - is this the pending link, waiting for Dev?

Reply:
confirmed

Reply:
not sure if waiting for a link about this breaking change, or, waiting for the linked Ref docs to be updated.
@victorhuangwq @peiche-jessica

Reply:
updated Ref docs content.  no additional link needed here